libs/helm/button/src/lib/hlm-button.ts

```diff
@@ -35,8 +36,9 @@ export const buttonVariants = cva(
 export type ButtonVariants = VariantProps<typeof buttonVariants>;
 
 @Directive({
-	selector: '[hlmBtn]',
+	selector: 'button[hlmBtn], a[hlmBtn]',
 	exportAs: 'hlmBtn',
+	hostDirectives: [{ directive: BrnButton, inputs: ['disabled', 'disabled'] }],
```

Comment:
Looks like the disabled input is registered twice here, I think we can remove the duplicate.